src/Tizen.Sensor/Tizen.Sensor/SensorEnumerations.cs

```diff
@@ -332,28 +332,34 @@ public enum DetectorState
     [EditorBrowsable(EditorBrowsableState.Never)]
     public enum AutoRotationState
     {
+        /// <summary>
+        /// Degree_Unknown state.
+        /// </summary>
+        /// <since_tizen> 3 </since_tizen>
+        Degree_Unknown = 0,
```

Comment:
I recommend to remove Unknown value.
you do better throw exception rather than return unknown value
or use Degree_0 as default value

Reply:
@myroot done!

Reply:
@myroot done!